cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/AppFabricServer.java

```diff
@@ -123,6 +134,16 @@ public AppFabricServer(CConfiguration configuration, DiscoveryService discoveryS
     this.pluginService = pluginService;
     this.privilegesFetcherProxyService = privilegesFetcherProxyService;
     this.routeStore = routeStore;
+    this.sslEnabled = configuration.getBoolean(Constants.Security.AppFabric.SSL_ENABLED);
```

Comment:
Will we add one configuration for each of the system services? Perhaps we should just use ```Constants.Security.SSL_ENABLED``` and enable it for all the services (Router, AppFabric, System Services)